qa/tests/features/new/manual/decision-tree/dataset-creation/dataset-creation.feature

```diff
@@ -0,0 +1,101 @@
+Feature: Decision Tree Panel, Create dataset
+	As the Anfisa user I want to create new datasets on the "Decision Tree Panel" page
+
+Background:
+	Given user opens the "Decision Tree Panel" for the "xl_PGP3140_wgs_NIST-4_2" dataset
+
+@regression
+@smoke
+Scenario: 01 Create dataset
+	Given "@Hearing Loss, v.5" decision tree was loaded
+	When clicks "Create Derive DS" button
+	And "Add new dataset" dialog is displayed
+	And user enters valid name for a dataset
+	And clicks "Add dataset" button
+	Then dataset should be created
```

Comment:
You should check also that you can open created dataset with "Open it" button and that created dataset has the correct number of variants.